schemas/1.21.0

```diff
@@ -50,6 +50,33 @@ versions:
         # https://github.com/open-telemetry/semantic-conventions/pull/53
         - rename_metrics:
             process.runtime.jvm.cpu.utilization: process.runtime.jvm.cpu.recent_utilization
+        # https://github.com/open-telemetry/opentelemetry-specification/pull/3464
+        - rename_attributes:
+            attribute_map:
+              memory: memory.type
+              pool: memory.pool.name
+            apply_to_metrics:
+              - metric.process.runtime.jvm.memory.usage
+              - metric.process.runtime.jvm.memory.init
+              - metric.process.runtime.jvm.memory.committed
+              - metric.process.runtime.jvm.memory.limit
+              - metric.process.runtime.jvm.memory.usage_after_last_gc
+        - rename_attributes:
+            attribute_map:
+              name: gc.name
+              action: gc.action
+            apply_to_metrics:
+              - process.runtime.jvm.gc.duration
+        - rename_attributes:
+            attribute_map:
+              daemon: thread.daemon
+            apply_to_metrics:
+              - process.runtime.jvm.threads.count
+        - rename_attributes:
+            attribute_map:
+              pool: buffer.pool.name
+            apply_to_metrics:
+              - attributes.process.runtime.jvm.buffer
```

Comment:
Is this correct?

Reply:
I think this is still wrong, no? @zeitlinger `attributes.process.runtime.jvm.buffer` is an attribute group, not a metric. This group applies to the metrics

`process.runtime.jvm.buffer.usage`, `process.runtime.jvm.buffer.limit` and `process.runtime.jvm.buffer.count`. 

Also: `apply_to_metrics`:

In the YAML definitions, we have the `id` (`metric.process.runtime.jvm.memory.usage`) but we also have the `metric_name` (`process.runtime.jvm.memory.usage`). If we are going to list them here, should the metric name or the id value be used? I think it's the metric name but I'm not sure. @tigrannajaryan maybe you know more?

Also: Do we want to list the metrics here, or just omit them? The [schema file format spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/schemas/file_format_v1.1.0.md#rename_attributes-transformation-2) says:

```
apply_to_metrics:
              # Optional. If it is missing the transformation is applied
              # to all metrics. If it is present the transformation is applied
              # only to the metrics with the name that is found in the sequence
              # specified below.
```